### ISSUE TITLE
JSON endpoint for thematic data url

### ISSUE THREAD

**Issue by mulfvik** [3dcitydb/3dcitydb-web-map#60]
It would be nice to have an json endpoint option for thematic data url. Quite like the "PostgreSQL REST API" option but expandable.

**Comment by thomashkolbe** [comment]
We are currently developing a generic "Data API" for the webclient, where different data sources can be added as plugins. It should be easy then to create an adapter for other JSON based services. However, it will take another 3-6 months before we will include it. The "Data API" will also allow writing and updating datasets (if the connected data source allows it).

**Comment by mulfvik** [comment]
Looking forward to the release, thank you 👍

**Comment by mulfvik** [comment]
Hi @thomashkolbe, I was wondering if this repo still is being developed or is it in hibernation mode?

**Comment by thomashkolbe** [comment]
Development was hibernated for quite some time, indeed. But there has been some progress.
@Son-HNguyen: would you please send more information?

**Comment by Son-HNguyen** [comment]
Hi @mulfvik and @thomashkolbe, we are currently updating the Cesium engine to allow for i3s layers. We are now supporting four types of layers in parallel: COLLADA/KML/glTF, Cesium 3D Tiles, i3s, and GeoJSON, and for each layer, 3 different sources of thematic data can be used: Google Spreadsheets, PostgREST, as well as embedded (thematic data that is already available in the visualization datasets). This brings us already to 12 combinations for testing and bug fixing. On top of that, we need to make sure that multiple layers can be displayed together, as well as the correct highlighting of objects when clicked / hovered on. Changes will first be uploaded soon to the branch https://github.com/3dcitydb/3dcitydb-web-map/tree/cesium-1.100, which will then after testing be merged into the main branch.

**Comment by mulfvik** [comment]
I was thinking about updating the Cesium engine in a PR if this repo was still alive... and maybee a fix for the highlighting issue. But it seems this is getting cared of so thank you for the reply.

**Comment by Son-HNguyen** [comment]
Hi @mulfvik, yes we are working on updating the Cesium engine and fixing the highlighting issues, as well as improving the display of thematic data of clicked objects.

If you have any ideas or suggestions to enhance the web client, please feel free to create a PR and we will look into it. We welcome and appreciate every contribution. Thanks again for the PR you posted last week, which is now already in the master branch.
